docs/guides/platform/manager/an-overview-of-the-cloud-manager/index.md

```diff
@@ -1,24 +1,24 @@
 ---
-slug: an-overview-of-the-linode-cloud-manager
-description: "This guide provides you with an overview of the Linode Cloud Manager and covers how to locate features within Cloud Manager, create Linodes and more."
+slug: an-overview-of-the-cloud-manager
+description: "This guide provides you with an overview of the Cloud Manager and covers how to locate features within Cloud Manager, create Linodes and more."
 keywords: ["classic manager","cloud manager","linode"]
 license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-aliases: ['/platform/manager/a-classic-to-cloud-manager-overview-guide/','/platform/manager/an-overview-of-the-linode-cloud-manager/']
+aliases: ['/platform/manager/a-classic-to-cloud-manager-overview-guide/','/platform/manager/an-overview-of-the-linode-cloud-manager/', '/platform/manager/an-overview-of-the-linode-cloud-manager/']
```

Comment:
The alias `/platform/manager/an-overview-of-the-linode-cloud-manager/` is repeated twice in the `aliases` array. Change to `/guides/an-overview-of-the-linode-cloud-manager/` to add an alias for the current URL to the new URL.